### ISSUE TITLE
Unable to book 2 beneficiaries together

### ISSUE THREAD

**Issue by networkrahul** [cowinapi/developer.cowin#309]
Earlier, after logging in, and clicking on Schedule, it allowed multiple select (we could click on schedule for multiple beneficiaries) and then it took to the search page. Now if I click on 1 beneficiary's schedule button, it directly takes to search page. If I book only for 1 beneficiary, then it will be very hard to get same slot for the other, given the less number of slots

**Comment by sarithdm** [comment]
> Earlier, after logging in, and clicking on Schedule, it allowed multiple select (we could click on schedule for multiple beneficiaries) and then it took to the search page. Now if I click on 1 beneficiary's schedule button, it directly takes to search page. If I book only for 1 beneficiary, then it will be very hard to get same slot for the other, given the less number of slots

Yes. This option will be useful as the slots are getting booked in less than a minute.

**Comment by rchawathearkin** [comment]
I think cowin developers have removed a very good feature. With captcha, it is practically impossible to book multiple (even 2) family members together in 1 go before the appointment slots get booked. Imagine the hassle of able to book 1 out of 2 family members at a time and having to go out multiple times to take the vaccine and expose yourself multiple times.

**Comment by riyaznet** [comment]
Yes, this feature is much needed in the Cowin web app. Cowin team is requested to bring it back.
